ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml

```diff
@@ -31,3 +31,48 @@ periodics:
     labels:
       master: ci.openshift.redhat.com
     name: azure_build_node_image_rhel_310
+- agent: kubernetes
+  cron: ""
+  decorate: true
+  interval: 24h
+  name: azure-build-node-image-rhel-310
+  skip_cloning: true
+  spec:
+    containers:
+    - args:
+      - --secret-dir=/usr/local/e2e-azure-secret
+      - --template=/usr/local/azure-image-build
+      - --target=azure-image-build
+      command:
+      - ci-operator
+      env:
+      - name: CONFIG_SPEC
+        valueFrom:
+          configMapKeyRef:
+            key: release-3.10.yaml
+            # Use openshift-ansible ci-operator config for as long as
+            # the node build code is part of that repo.
+            name: ci-operator-openshift-openshift-ansible
+      - name: JOB_NAME_SAFE
+        value: azure-image-build
+      - name: CLUSTER_TYPE
+        value: azure
```

Comment:
One of the first thing we will need to enhance is to add variables to be "rhel or centos" . Im making assumption ci-operator will pick those up same as TEST_FOCUS ones here:
https://github.com/openshift/release/blob/master/ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-presubmits-release-3.11.yaml#L124

Reply:
Follow-up issue

Reply:
https://github.com/openshift/openshift-azure/issues/439